### PR TITLE
feat: White background for Board component

### PR DESCRIPTION
**Pull request by gabalafou**
Irina asked me to change the container for the consulting services grid on the home page so that the background is white. 

I haven't been able to test it locally yet, but I think these are the needed changes.

### CODE REVIEW

**Comment by gabalafou** [comment]
Hmm. I don't see draft mode. Well, just be aware there's an incoming change

File: apps/consulting/components/Board/Board.tsx [hunk 1]
```diff
@@ -14,22 +16,43 @@ export const Board: FC<TBoardProps> = ({
...
+        <h2 className="text-[4rem] font-extrabold tracking-[0.02em] leading-[5rem] sm:text-[4.8rem] text-violet font-heading">
           {title}
         </h2>
         <div
           className="flex flex-col gap-8 mt-[2.8rem] mr-[5.5rem] mb-[4.3rem] text-[1.8rem] leading-[2.2rem] sm:gap-12 sm:mt-28 sm:mr-0 sm:mb-[6rem] lg:mx-36  xl:mt-[3.6rem]"
           dangerouslySetInnerHTML={createMarkup(description)}
         />
         <div className="grid grid-cols-1 sm:grid-cols-3 sm:gap-[0.5px] sm:bg-white xl:gap-[1px]">
-          {grid.map((props) => (
-            <BoardItem {...props} key={props._uid} />
```
Comment (gabalafou) [inline apps/consulting/components/Board/Board.tsx:30]: Originally I tried wrapping these with a div that had the border styles, like so:

    <div className={classNameBorder}><BoardButton {...button} /></div>

But that messed up the alignment of items in the BoardButton's grid square, so I u

**Comment by marcin-antczak-naturaily** [comment]
@gabalafou would you be so kind and resolve conflicts?

**Comment by gabalafou** [comment]
@marcin-antczak-naturaily, done. Resolved.

**Comment by gabalafou** [comment]
@marcin-antczak-naturaily do you want me to merge this is now?

**Pull request merged**